test/functional/deprecated/iDataQueueFunctional.js

```diff
@@ -44,21 +44,39 @@ if (config.transport === 'rest') {
 
 const lib = 'NODETKTEST'; const dqName = 'TESTQ';
 
+let deprecation = null;
+function deprecationHandler(dep) {
+  deprecation = dep;
+}
+
+function getDeprecation() {
+  const temp = deprecation;
+  deprecation = null;
+  return temp;
```

Comment:
Do we have to worry about race conditions here? Can mocha run tests simultaneously?

Reply:
By default mocha runs test serially https://mochajs.org/#serial-mode. So in step 12 it describes the flow of execution:
![image](https://user-images.githubusercontent.com/33973272/96922992-41594d00-1476-11eb-8d15-76f8eb05b9df.png)

So I do not think there would be issue of tests running simultaneously unless we were using parrell mode https://mochajs.org/#parallel-mode